lib/serializer/Connection.h

```diff
@@ -79,17 +79,17 @@ class DLL_LINKAGE CConnection
 	std::shared_ptr<boost::mutex> mutexRead;
 	std::shared_ptr<boost::mutex> mutexWrite;
 	std::shared_ptr<TSocket> socket;
-	bool connected;
-	bool myEndianess, contactEndianess; //true if little endian, if endianness is different we'll have to revert received multi-byte vars
+	bool connected = false;
```

Comment:
Please, keep initialization of all members in one place, either in constructor or in-place.
Don't want to play "find where this field is initialized" game.

Reply:
@rilian-la-te I think you missed this comment?

Reply:
done. This was clang-tidy bogus initialization.